README.md

```diff
@@ -5,21 +5,33 @@
 [![nightly-build](https://github.com/ProjectPythia/pythia-cookoff-2024/actions/workflows/nightly-build.yaml/badge.svg)](https://github.com/ProjectPythia/pythia-cookoff-2024/actions/workflows/nightly-build.yaml)
 
 
-## Save the Date!
+## Register Now!
 
-**June 11-14 2024, in person at the [NCAR Mesa Lab](https://scied.ucar.edu/visit) and virtual**
+Registration for the 2024 Pythia Cook-off hackathon for geoscientific Python Cookbook development is now live!
+
+Join Project Pythia June 11-14 2024 in-person in Boulder, CO at the [NCAR Mesa Lab](https://scied.ucar.edu/visit) or virtually via Zoom.
+
+New in 2024, Project Pythia has limited funding from the U.S. National Science Foundation to support travel expenses for in-person participants!
+If full or partial travel support could help you attend this year's hackathon, please see the [](travel.md) page for more information.
+
+The deadline for registration is **May 26**, with priority registration for travel support consideration due by **March 29**. There is a $50 USD registration fee due at the time of registration to support event expenses.
```

Comment:
Are we charging a registration fee for virtual attendees?

Reply:
Oh good point. I'll clarify